src/content/structured/components/side-nav/code.mdx

```diff
@@ -747,6 +749,115 @@ export const withReactRouter = [
   </MemoryRouter>
 </ComponentPreview>
 
+### With MUI icons in SlottedSVG
+
+export const withMUIicons = [
+  {
+    language: "React",
+    snippet: `<MemoryRouter initialEntries={["/"]}>
+  <div style={{ display: "flex", flexDirection: "row"}}>
+    <IcSideNavigation
+      appTitle="ACME"
+      version="v0.0.0"
+      status="BETA"
+    >
+      <SlottedSVG slot="app-icon" ... />
```

Comment:
The `SlottedSVG` elements are not showing how to use MUI Icons. The code snippet and working example should be identical in this case.